### ISSUE TITLE
Error when fetching notes for deleted issue

### ISSUE THREAD

**Issue by adcox** [piceaTech/node-gitlab-2-github#43]
The problem: an error reading
```
Could not fetch notes for GitLab issue #undefined
{ StatusCodeError: 400 - {"error":"noteable_id is invalid"}
```

To recreate:
* Migrate a project with a deleted issue

The problem is that the[ `placeholderIssue`](https://github.com/piceaTech/node-gitlab-2-github/blob/a8f57d843b715395c63fc370415310e3aef925a1/index.js#L235) does not include the `iid` field. When the `createIssueComments` function attempts to [fetch the issue notes](https://github.com/piceaTech/node-gitlab-2-github/blob/a8f57d843b715395c63fc370415310e3aef925a1/index.js#L529) for such a placeholder, `issue.iid` is undefined and the error above is thrown.

**Comment by spruce** [comment]
This is fixed by #44 ?

**Comment by adcox** [comment]
Yes - I should have included that in the pull request description.
